pkg/util/util.go

```diff
@@ -904,9 +904,6 @@ func GetAndExtractZip(zipURL string, destination string, pathToUnzip string) err
 	if zipURL == "" {
 		return errors.Errorf("Empty zip url: %s", zipURL)
 	}
-	if !strings.Contains(zipURL, ".zip") {
-		return errors.Errorf("Invalid zip url: %s", zipURL)
-	}
 
 	var pathToZip string
 	if strings.HasPrefix(zipURL, "file://") {
```

Comment:
I am ok with this PR but do we want to also enforce that the url starts with `file://` a url could be a simple http or https one but could have a zip file returned?

Reply:
You're missing the rest of the code which deals with that… 😉